view/frontend/templates/callback_verify.phtml

```diff
@@ -7,5 +7,4 @@
  * file that was distributed with this source code.
  */
 ?>
-
-<?= $this->escapeHtml($this->getMessage());
+<?= $block->escapeHtml($block->getMessage()); ?>
```

Comment:
Instead of turning PHP on an off multiple times maybe just use:
`echo $block->escapeHtml($block->getMessage());`

Reply:
Magento suggests a short echo tag, if we change as above Magento codesniffer show this warning :
`"Short echo tag syntax must be used; expected "<?=" but found "<?php echo"`